prboom2/src/m_menu.c

```diff
@@ -2720,7 +2720,6 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
 setup_menu_t stat_settings2[] =
 {
   {"ADVANCED HUD SETTINGS"       ,S_SKIP|S_TITLE,m_null,ADVHUD_X,SB_Y+1*8},
-  {"REPORT REVEALED SECRETS"     ,S_YESNO     ,m_null,ADVHUD_X,SB_Y+ 2*8, {"hudadd_secretarea"}},
```

Comment:
You'll need to adjust the indices of the options below this one to shift them up to fill the gap if we remove from here.

Reply:
Yeah... I realized this just now while I was reviewing the changes. _I had one job._

I accidentally left an empty white space there now, damn. Not sure what to make of it though, it seems some aren't fond of this change, so I'd rather just close this and call it a day.